FsSonarRunner/pom.xml

```diff
@@ -35,7 +35,7 @@
               </arguments>
             </configuration>
           </execution>
-          <execution>
+          <!-- <execution>
```

Comment:
`dotnet publish --no-build` will complain on missing files, thus it is not help to do the build separately